### DISCUSSION TITLE
Physics time scale

### DISCUSSION THREAD

**Discussion by makcimbx** [FirstGearGames/FishNet#596] [Approved Changes or Features]
Add PhysicsTimeScale, that can be managed in TimeManager that will multiply tick delta everywhere. Quite a handy feature that you now have to add yourself after every update. Something like this:
`Physics.Simulate(tickDelta * _networkManager.TimeManager.PhysicsTimeScale);`

**Comment by FirstGearGames** [comment]
Doable but would cause desync if clients didnt use the same scale during same ticks.

**Reply by makcimbx** [reply]
Sync value on clients when connect to server?

**Comment by FirstGearGames** [comment]
I think this could be a good feature and also fairly easy to add in with no risk. This has been accepted and made a priority.

**Comment by FirstGearGames** [comment]
4.2.1 was just released, I'll get this in next release!

**Comment by FirstGearGames** [comment]
Added in 4.2.3.
